app/src/main/java/com/futurice/freesound/app/module/ApiModule.java

```diff
@@ -55,16 +56,14 @@
 
     @Provides
     @Singleton
-    FreeSoundApi provideFreeSoundApi(Endpoint endpoint,
-                                     Client client,
-                                     Converter converter,
-                                     RequestInterceptor requestInterceptor) {
-        return new RestAdapter.Builder()
-                .setEndpoint(endpoint)
-                .setClient(client)
-                .setConverter(converter)
-                .setRequestInterceptor(requestInterceptor)
-                .setLogLevel(RestAdapter.LogLevel.FULL)
+    FreeSoundApi provideFreeSoundApi(@Named(URL_CONFIG) String url,
```

Comment:
Make the `@Provides` methods these static if they can be. Rebasing will resolve most of these.